charts/webredirect/templates/service.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if eq .Values.ingress.destination.backendService.createExternalname true -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "webredirect.fullname" . }}
+  labels:
+    {{- include "webredirect.labels" . | nindent 4 }}
+spec:
+  type: ExternalName
```

Comment:
I should add that I've configured the optional "existing backendService" (at `.ingress.destination.backendService`), because I've encountered issues with the fact that we've got several redirects that point to the same host, like:
```
anynines.com -> https://www.anynines.com
data-services.anynines.com -> https://www.anynines.com/de/data_services
```
But now that I think about it, it doesn't make any sense to create this dependency on other services, because we anyways have different names for the ingresses and we can use the same for the services.